src/pages/blog/starlingx-deploying-your-app-1.md

```diff
@@ -0,0 +1,330 @@
+---
+templateKey: blog-post
+title: Deploying your application to StarlingX
+author: Bruno Muniz
+date: 2023-10-25T12:00:00.042Z
+category:
+  - label: Features & Updates
+---
+
+Deploying your own application to StarlingX is as easy as any other Kubernetes
+deployment out there. In this blog post I will show you the process with a
+simple demo app.<!- more ->
+
+It's important to understand that an application can be deployed in many ways
+on the [Kubernetes cluster(s) that StarlingX manages](https://docs.starlingx.io/operations/k8s_cluster.html)
+:
+
+- [with plain Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/) methods;
+- with [Helm](https://helm.sh/docs/intro/using_helm/#helm-install-installing-a-package);
+- with [Flux](https://fluxcd.io/); and finally
+- as a StarlingX Application, which benefits from tight integration with the
+  [StarlingX system](https://opendev.org/starlingx/config).
+
+In this article I will focus on [Helm](https://helm.sh/),
+which is the most popular package manager for Kubernetes. Future blog posts
+will address other deployment types and their characteristics.
+
+I will use a virtual All-In-One Simplex (AIO-SX) setup of StarlingX. If you
+want to follow along, you can install your own by following the related 
+[StarlingX install guide](https://docs.starlingx.io/deploy_install_guides/release/virtual/automated_install.html#dashboards).
+
+Anyone who already packages their application with
+[Helm](https://helm.sh/) will be able to deploy it on StarlingX without additional hassle.
+
+## The Demo App
+
+I will use a simple demo app for this blog post. We won't
+dive into the use case of the demo app, but there will be
+additional information about it at the end of the post.
+
+Now, let's focus on deploying the app!
```

Comment:
This suggestion is just a complement of the suggestion above so they can be merged together from the GH review itself.

```suggestion
```